templates/csharp/ai-bot/AIBotApplication.cs.tpl

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using Microsoft.TeamsAI;
+using Microsoft.TeamsAI.State;
+
+namespace {{SafeProjectName}};
+
+public class AIBotApplication : Application<TurnState, TurnStateManager>
+{
+    public AIBotApplication(ApplicationOptions<TurnState, TurnStateManager> options) : base(options) { 
+        AI.ImportActions(this);
+    }
+
+    protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivity> turnContext, TurnState turnState, CancellationToken cancellationToken)
```

Comment:
Remove this activity handler to sync with js template.

Reply:
Done.